views/layouts/_head.html.haml

```diff
@@ -0,0 +1,15 @@
+%head
+  %meta{ content: "text/html; charset=UTF-8", "http-equiv" => "Content-Type" }/
+  %meta{ charset: "utf-8" }/
+  %meta{ content: "width=device-width, initial-scale=1, shrink-to-fit=no", name: "viewport" }/
+  %title EGI Marketplace
+  = csrf_meta_tags
+  = csp_meta_tag
+  = favicon_pack_tag "favicon.ico"
+  = favicon_pack_tag "favicon-180x180.png", rel: "apple-touch-icon", type: "image/png"
+  = stylesheet_pack_tag "application"
+  = javascript_pack_tag "application"
+  - if Rails.env.production?
+    = render "layouts/matomo"
```

Comment:
I'm not sure if we should provide matomo tracking and generally our trackers to these instances. @bwilk, @agpul ?

Reply:
You can comment or remove everything from line 12 to line 15 and I'm not sure if there is a lack of free line at the end